Vanilla/Resources/dictionary.js

```diff
@@ -12,6 +13,7 @@ let present = [{
 	prefix: "it is"
 }];
 
+// Weather API codes for forecast conditions
```

Comment:
Here again, for forecast conditions, it makes a lot of sense to add this comment.